exercise/viewbase.py

```diff
@@ -186,9 +189,11 @@ def get_resource_objects(self) -> None:
 
 class SubmissionMixin(SubmissionBaseMixin, ExerciseMixin):
 
-    def get_submission_object(self):
+    def get_submission_object(self) -> Submission:
         return get_object_or_404(
-            Submission,
+            Submission.objects.prefetch_related(None).prefetch_related(
+                models.Prefetch('submitters', UserProfile.objects.prefetch_tags(self.instance))
+            ),
```

Comment:
Yes, SubmissionManager already prefetches the submitters without their usertags, and you can't call `prefetch_related` on the same relationship twice without resetting it like that.

I guess I could've changed SubmissionManager to prefetch submitters with their tags, but I didn't feel like it was appropriate because they aren't _always_ needed.